oe_webtools.libraries.yml

```diff
@@ -3,3 +3,8 @@ drupal.webtools-smartloader:
   version: 1.x
   js:
     https://europa.eu/webtools/load.js: { type: external, attributes: { defer: true } }
+
+oe_webtools.component-collect:
```

Comment:
We don't seem to be using this library?

Reply:
@imanoleguskiza we will in EWCMS, and it's generic. Used in cases when we need to add aa webtools widget as part of an ajax request.